modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -99,10 +99,18 @@ def default_headers
     end
 
     def claims_default_header
```

Comment:
Can we please have a few unit tests covering these changes?

Reply:
added in 99725f52ffe15cdbbd7a4b5f8ffa31c46e78c28a